bc_obps/reporting/models/report_product.py

```diff
@@ -0,0 +1,74 @@
+from django.db import models
+from registration.models.regulated_product import RegulatedProduct
+from registration.models.time_stamped_model import TimeStampedModel
+from reporting.models.facility_report import FacilityReport
+from reporting.models.report_version import ReportVersion
+
+
+class ReportProduct(TimeStampedModel):
+    """
+    A model storing production information for a single product.
+    It belongs to a facility report.
+    """
+
+    report_version = models.ForeignKey(
+        ReportVersion,
+        on_delete=models.CASCADE,
+        related_name="report_products",
+        db_comment="The report this production information relates to",
+    )
+
+    facility_report = models.ForeignKey(
+        FacilityReport,
+        on_delete=models.CASCADE,
+        related_name='report_products',
+        db_comment="The facility report this production information belongs to",
+    )
+
+    product = models.ForeignKey(
+        RegulatedProduct,
+        on_delete=models.PROTECT,
+        related_name='report_products',
+        db_comment="The product this production information is about",
+    )
+    annual_production = models.FloatField(
```

Comment:
Question: Should we consider using DecimalField here? Or does the slight imprecision of floats not matter with these values? They will be used in compliance calculations later

Reply:
@dleard after thinking about it, I would suggest leaving it as a float and bringing this to the BA. Decimal fields constrain how many decimal places we can accomodate, what if some products are in small quantity (I report 0.0034723 units) and others are very large quantities?
Floats will let us handle that for now, and we can change it later before release if we get more precise requirements